ui/app/templates/clients/client.hbs

```diff
@@ -101,7 +101,7 @@
               <th class="is-narrow"></th>
               {{#t.sort-by prop="shortId"}}ID{{/t.sort-by}}
               {{#t.sort-by prop="modifyIndex" title="Modify Index"}}Modified{{/t.sort-by}}
-              {{#t.sort-by prop="name"}}Name{{/t.sort-by}}
+              {{#t.sort-by prop="createIndex" title="Create Index"}}Created{{/t.sort-by}}
```

Comment:
Not a lot of background around Nomad specifically, are names not useful for folks?

Reply:
Not really. Names for allocations are a function of task group name and count. They end up looking like `task-group-name.[3]`.